tools/train.py

```diff
@@ -193,12 +202,14 @@ def main():
         # save mmdet version, config file content and class names in
         # checkpoints as meta data
         cfg.checkpoint_config.meta = dict(
-            mmdet_version=__version__,
+            mmdet3d_version=__version__,
```

Comment:
We may also add the version of mmdet & mmseg

Reply:
Add.